extensions/blocks/contact-form/index.js

```diff
@@ -69,7 +69,7 @@ export const settings = {
 	},
 
 	edit: JetpackContactForm,
-	save: InnerBlocks.Content,
+	save: function() { return ( <InnerBlocks.Content /> ); },
```

Comment:
Seeing same behavior here with function vs fat arrow, so this gets past the validation check, but there are regressions in behavior.